app/screens/error_teams_list/error_teams_list.js

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {
+    InteractionManager,
+    View,
+} from 'react-native';
+
+import FailedNetworkAction from 'app/components/failed_network_action';
+import Loading from 'app/components/loading';
+import StatusBar from 'app/components/status_bar';
+import {makeStyleSheetFromTheme, setNavigatorStyles} from 'app/utils/theme';
+
+const errorTitle = {
+    id: 'error.team_not_found.title',
+    defaultMessage: 'Team Not Found',
+};
+
+const errorDescription = {
+    id: 'mobile.failed_network_action.shortDescription',
+    defaultMessage: 'Make sure you have an active connection and try again.',
+};
+
+export default class ErrorTeamsList extends PureComponent {
+    static propTypes = {
+        actions: PropTypes.shape({
+            loadMe: PropTypes.func.isRequired,
+            connection: PropTypes.func.isRequired,
+            logout: PropTypes.func.isRequired,
+            selectDefaultTeam: PropTypes.func.isRequired,
+        }).isRequired,
+        navigator: PropTypes.object,
+        theme: PropTypes.object,
+    };
+
+    constructor(props) {
+        super(props);
+        props.navigator.setOnNavigatorEvent(this.onNavigatorEvent);
+
+        this.state = {
+            loading: false,
+        };
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (this.props.theme !== nextProps.theme) {
+            setNavigatorStyles(this.props.navigator, nextProps.theme);
+        }
+    }
+
+    goToChannelView = () => {
+        const {navigator, theme} = this.props;
+
+        navigator.resetTo({
+            screen: 'Channel',
+            animated: false,
+            navigatorStyle: {
+                navBarHidden: true,
+                statusBarHidden: false,
+                statusBarHideWithNavBar: false,
+                screenBackgroundColor: theme.centerChannelBg,
+            },
+        });
+    };
+
+    getUserInfo = async () => {
+        this.setState({loading: true});
+        this.props.actions.connection(true);
```

Comment:
There are chances that this is set false so network requests wont go through so manually setting this to `true`